pkg/detectors/privatekey/cracker_test.go

```diff
@@ -68,11 +68,15 @@ func Test_crack(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			_, err := crack(tt.in)
+			_, passphrase, err := crack(tt.in)
 			if (err != nil) != tt.wantErr {
 				t.Errorf("crack() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
+
+			if passphrase != string(testEncryptedKeyCorrectPassword) {
+				t.Errorf("crack() passphrase = %v, want %v", passphrase, string(testEncryptedKeyCorrectPassword))
+			}
```

Comment:
I would expect this new check to require updating the test case itself. Does that need to be done?

Reply:
It didn't, but I've added the wanted passphrase as a parameter for if we add more cases.